habitat-baselines/human_controllers/amass_human_controller.py

```diff
@@ -0,0 +1,593 @@
+# Copyright (c) Meta Platforms, Inc. and its affiliates.
```

Comment:
A lot of new code here, I hope we can get some good comments and discussion here before merging. One drawback with multi-thousand-line PRs like this one is that most of us reviewers are probably hitting fatigue by the time we get down to these files.